react/features/subtitles/components/TranscriptionSubtitles.web.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * The type of the React {@code Component} props of
+ * {@link TranscriptionSubtitles}.
+ */
+type Props = {
+
+    /**
+     * Map of transcriptMessageID's with corresponding transcriptMessage.
+     */
+    transcriptMessages: Map<string, Object>
```

Comment:
jitsi-meet has a convention where state retrieved from redux directly from mapStateToProps is prefixed with an underscore.